api/services/cardService.js

```diff
@@ -16,9 +16,8 @@ module.exports = {
             }
             return resolve(card);
           });
-      } else {
-        return Promise.reject({ message: 'Invalid arguments for findCard' });
       }
+      return Promise.reject({ message: 'Invalid arguments for findCard' });
```

Comment:
This is going to conflict with the changes in https://github.com/cuttle-cards/cuttle/pull/161/files#diff-90d500bee1581144835b0ef79d6747a0359efdb86c692d69f6ccc1919c387563R20

Reply:
I don't think it will since they're both making the same (linted) change.  If it does though, I'll fix it in the second MR before that's merged in